libs/application/templates/general-fishing-license/src/utils/licenses.ts

```diff
@@ -0,0 +1,47 @@
+import { FishingLicenseEnum } from '../types'
+
+export const FILE_UPLOAD_ACCEPT = '.pdf, .jpg, .jpeg, .png'
+
+// Determines whether fishing license has a file upload field
+export const licenseHasFileUploadField = (
+  license: FishingLicenseEnum | string,
+) => {
+  return (
+    license === FishingLicenseEnum.FREETIME ||
+    license === FishingLicenseEnum.FREETIMEHOOK ||
+    license === FishingLicenseEnum.FREETIMEHOOKMED ||
+    license === FishingLicenseEnum.OCEANQUAHOGIN ||
+    license === FishingLicenseEnum.COMMONWHELK ||
+    license === FishingLicenseEnum.CRUSTACEANS
+  )
+}
+
+// Determines whether fishing license has a dropdown to select area (veiðisvæði)
+export const licenseHasAreaSelection = (
+  license: FishingLicenseEnum | string,
+) => {
+  return license === FishingLicenseEnum.GREYSLEPP
+}
+
+// Determines whether fishing license has field to input number of roe nets (fjöldi hrognkelsaneta)
+export const licenseHasRailNetAndRoeNetField = (
+  license: FishingLicenseEnum | string,
+) => {
+  return license === FishingLicenseEnum.GREYSLEPP
+}
+
+export const MAXIMUM_TOTAL_RAIL_NET_LENGTH = 7500
+
+// Calculates the total sum of rails depending on the number of roe nets
+// To show the calculated result in front end or to check input for validation
+// Takes in a string to support text input
+export const calculateTotalRailNet = (roenet?: string, railnet?: string) => {
```

Comment:
We would probably benefit from having a unit test to verify the functionality.

Reply:
I dont see any unit tests in GFL project. Can you tell me where they are usually placed in the project?